src/aap_eda/tasks/orchestrator.py

```diff
@@ -432,6 +475,7 @@ def start_rulebook_process(
         ActivationRequest.START,
         request_id,
     )
+    monitor_rulebook_processes.delay()
```

Comment:
Do we really need to start a monitor immediately after each user request? Although it does hurt, it also won't help since the request just gets to be added to the internal queue.

Reply:
This is the same approach that we already took in the weboscket. It will be dispatched in the monitor, but scheduling it may accelerate the process. In the best case we gain 5 seconds, but I think it does not hurt either. 
Do you want to remove it?

Reply:
Sorry. My original comment meant "it does *not* hurt". I am thinking by the time the monitor is running the task hasn't finished or even started yet. Sure it does not hurt.

Reply:
>I am thinking by the time the monitor is running the task hasn't finished or even started yet. Sure it does not hurt.
Well, that's where the lock takes place. Anyway it would not be different from what we are already doing it in https://github.com/ansible/eda-server/pull/1262/files#diff-179935aa48d00084c8f2722f9b007cad6d09b24425ffe4121cc75f6c37b03c42R214

I will keep it if you agree.